modules/ROOT/pages/dataweave-streaming.adoc

```diff
@@ -161,7 +161,7 @@ Given this XML source data, you can set the unit of the stream to `<order-item/>
 </flow>
 ----
 
-Note that you need to set both `streaming=true` and the `collectionPath` value. If either  is missing, DataWeave will not stream the content.
+Note that you need to set both `streaming=true` and the `collectionPath` value. If either  is missing, DataWeave won't stream the content.
```

Comment:
Typically we don't use future tense, so "doesn't" is more standard here. (Just commenting in this PR, but true throughout.)